piccolo/apps/asgi/commands/templates/app/_xpresso_app.py.jinja

```diff
@@ -0,0 +1,104 @@
+import typing as t
+
+from piccolo.engine import engine_finder
+from piccolo.utils.pydantic import create_pydantic_model
+from piccolo_admin.endpoints import create_admin
+from starlette.staticfiles import StaticFiles
+from xpresso import App, FromJson, FromPath, HTTPException, Operation, Path
+from xpresso.routing.mount import Mount
+
+from home.endpoints import home
+from home.piccolo_app import APP_CONFIG
+from home.tables import Task
+
+TaskModelIn: t.Any = create_pydantic_model(
+    table=Task, model_name="TaskModelIn"
+)
+TaskModelOut: t.Any = create_pydantic_model(
+    table=Task, include_default_columns=True, model_name="TaskModelOut"
+)
+
+
+async def tasks() -> t.List[TaskModelOut]:
+    return await Task.select().order_by(Task.id).run()
```

Comment:
Can we change these to drop the `run`? So it's just `await Task.select().order_by(Task.id)`.

I know the other ASGI templates use `run`, but I'd like to update them so they're consistent with the docs.